chapters/tutorials/index.rst

```diff
@@ -0,0 +1,403 @@
+
+At the end of this tutorial, you will have your own chatbot. You will use text as an input and get a text result as output.
+For this example, we will use a `covid dataset <https://www.kaggle.com/xhlulu/covidqa>`_.
+You will understand how every part of this example works and how you could create new apps with different datasets on your own.
+
+Set-up & overview
+----------------------------------
+
+We recommend creating a `new python virtual environment <https://docs.python.org/3/tutorial/venv.html>`_, to have a clean install of Jina and prevent dependency clashing.
+
+We can start by installing Jina:
+
+  .. code-block:: python
+
+    pip install jina
+
+For more information on installing Jina, refer to this `page <https://docs.jina.ai/chapters/install/os/via-pip>`_.
+
+And we need the following dependencies:
+
+  .. code-block:: python
+
+    pip install click==7.1.2
+    pip install transformers==4.1.1
+    pip install torch==1.7.1
+
+
+Once you have Jina and the dependencies installed, let's take a broad overview of what we should do:
+
+.. image:: res/flow.png
+   :width: 600
+
+At the beginning of the Flow, you have your data, and this can be any type:
+
+* Audio
+* Image
+* Video
+* Text
+* Or any other type
+
+In this case, we are using text, so in the image, you see text only. But it can be whatever type you want.
+
+Once we have our data, as usual in Machine Learning, it's probable that you might need to pre-process that data. To keep this as simple as possible for this first tutorial, we won't need to do any pre-processing. But remember that this is a possibility for another use case.
+Once that is ready, we can encode our data into vectors and finally store those vectors, so it is ready for indexing and querying.
+
+Tutorial
+---------
+
+Define data and work directory
+++++++++++++++++++++++++++++++++++++
+
+We can start creating an empty folder, I'll call mine `tutorial` and that's the name you'll see through the tutorial but feel free to use whatever you wish.
+
+We will display our results in our browser, so download the `static` folder from `here <https://github.com/jina-ai/jina/tree/master/jina/helloworld/chatbot/static>`_, and paste it in in your `tutorial` folder. This is only the CSS and HTML files to render our results. We will use a dataset in a .csv format. I'll use the `COVID <https://www.kaggle.com/xhlulu/covidqa>`_ dataset from Kaggle. You don't need to download this by hand, we'll do it later in our app.
+
+Create a Flow
+++++++++++++++++++++++++++++++++++++
+
+To create a Flow you only need to import it from Jina:
+
+.. code-block:: python
+
+    from jina import Flow
+    f = Flow()
+
+But this is an empty Flow, since we want to encode our data and then index it, we need to add elements to it.
```

Comment:
But this is an empty Flow. Since we want to encode our data and then index it, we need to add Executors to it.

- Note I broke sentence
- I changed elements to Executors. Should probably link that to Executor cookbook